README.md

```diff
@@ -73,6 +75,10 @@ to the files you want to process JSON and set the necessary switches to enable C
 
 You can further use file [`include/nlohmann/json_fwd.hpp`](https://github.com/nlohmann/json/blob/develop/include/nlohmann/json_fwd.hpp) for forward-declarations. The installation of json_fwd.hpp (as part of cmake's install step), can be achieved by setting `-DJSON_MultipleHeaders=ON`.
 
+### Pkg-Config
```

Comment:
Please move this paragraph to the "Meson" entry in the "Package Managers" section.

Reply:
It should be distinct from meson, because you can use the pkg-config file from a Makefile too, not just from a meson.build file -- but sure, it can be moved to the package managers section.

Reply:
This paragraph is empty now, right?

Reply:
Oops.